chainerx_cc/chainerx/stack_vector.h

```diff
@@ -194,4 +195,33 @@ class StackVector {
     BaseContainer d_{};
 };
 
+// Formatter to print StackVector containing integral elements as e.g. '[]' or '[1, 2, 3]'.
+template <typename T, stack_vector_detail::size_type N>
+class DimArgFormatter {
```

Comment:
How about `DimsArgFormatter`? A single argument holds multiple dimensions.

Reply:
Okay, will fix.

Reply:
Ah, that was also you suggestion. Changed to it.